drivers/SmartThings/zwave-bulb/profiles/rgbw-bulb-2580K-7050K.yml

```diff
@@ -12,6 +12,10 @@ components:
           range: [1, 100]
   - id: colorTemperature
     version: 1
+    config:
+      values:
+        - key: "colorTemperature.value"
+          range: [ 2580, 7050 ]
   - id: colorControl
     version: 1
   - id: refresh
```

Comment:
Should the metadata in these profiles be deleted now?

Reply:
I think so, yes.

Reply:
removed here and in other profiles